net/znc/Makefile

```diff
@@ -277,7 +278,7 @@ CONFIGURE_VARS += \
 	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
-	--disable-charset \
+	$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset))
```

Comment:
Apologies, I trusted the copy paste improvement. Noted.

On 3 June 2016 at 13:29, Jonas Gorski <notifications@github.com> wrote:

> In net/znc/Makefile
> <https://github.com/openwrt/packages/pull/2786#discussion_r65693312>:
>
> > @@ -277,7 +278,7 @@ CONFIGURE_VARS += \
> >  	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
> >
> >  CONFIGURE_ARGS += \
> > -	--disable-charset \
> > +	$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset))
>
> This line breaks the package completely, your dropped the \ at the end
> causing parse failures, and have a superfluous ) at the end of the line,
> causing compilation to fail after adding the .
>
> Please always test your changes before submitting.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openwrt/packages/pull/2786/files/9c1e7d620e60fabb410dc23629ffc8a3082cce97#r65693312>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANL6QJMYqf9avYpoQXbmcH8LvFqJcaOks5qIBAugaJpZM4IqrIW>
> .
>